python/graphstorm/eval/evaluator.py

```diff
@@ -584,7 +964,7 @@ def compute_score(self, pred, labels, train=True):
         return results
 
 class GSgnnLPEvaluator():
-    """ Template class for user defined evaluator.
+    """ Base class for user defined Link Prediction evaluator.
```

Comment:
Shouldn't GSgnnLPEvaluator inherit from GSgnnBaseEvaluator?

Reply:
Yes, will create a new GSgnnLinkPredictionEvaluator to replace the GSgnnLPEvaluator in the next PR.